pandas/io/pytables.py

```diff
@@ -3944,29 +3944,7 @@ def read(self, where=None, columns=None, **kwargs):
                 objs.append(obj)
 
         else:
-            warnings.warn(duplicate_doc, DuplicateWarning, stacklevel=5)
```

Comment:
@jreback this is the part of this PR (and the larger process) that I'm least confident about.  Are we sure we want to rip this out entirely or should it somehow be adapted to not use Panel?